webpack.config.js

```diff
@@ -23,5 +24,10 @@ module.exports = {
 		libraryTarget: 'commonjs2'
 	},
 	target: 'node',
-	externals: [nodeExternals()]
+	externals: [nodeExternals()],
+	plugins: [
+		new webpack.DefinePlugin({
+			PACKAGED: true
+		})
```

Comment:
@vitaly-t We are not using webpack for release of the complete app. Webpack is used just for the web socket layer. Notice:

```
entry: './workersController.js',
```

So I believe this `webpack.DefinePlugin` would not have impact or usage in reset of the application. 

For rest of the app we are using

```
obfuscator: {
			files: files,
			entry: 'app.js',
			out: 'release/app.js',
			strings: true,
			root: __dirname
		},
```

If we can convert that step also within webpack flow then the desired impact can be achieved as described in #1323